src/main/jupyter/oss/security/commons.py

```diff
@@ -129,3 +15,42 @@ def draw_hists(columns, test_vectors, width=20, height=5):
         test_vectors[column].value_counts().plot(kind='bar', rot=0, ax=axes[i])
         axes[i].set_title(textwrap.TextWrapper(width=25).fill(column))
         i = i + 1
+      
+    
+# load a list of test vectors from a YAML file
+def load_test_vectors_from_yaml(file, features):
+    rows = []
+    with open(file, 'r') as f:
+        raw_data = yaml.load(f)
+        for raw in raw_data:
+            data = {}
+            data['alias'] = raw['alias']
+            for value in raw['values']:
+                feature_type = value['type']
+                if feature_type == 'ScoreValue':
+                    feature_value = value['value']
+                elif feature_type == 'BooleanValue':
+                    feature_value = value['flag']
+                elif feature_type == 'DateValue':
+                    feature_value = value['date']
+                elif feature_type == 'IntegerValue' or feature_type == 'PositiveIntegerValue':
+                    feature_value = value['number']
+                elif feature_type == 'LgtmGradeValue':
+                    feature_value = value['value']
+                elif feature_type == 'UnknownValue':
+                    feature_value = 'unknown'
+                elif feature_type == 'VulnerabilitiesValue':
+                    feature_value = '...'
```

Comment:
is this a place holder?

Reply:
Yes. For complex type such as `VulnerabilitiesValue`, the function returns "...". Currently I don't know a good way how to visualise info about vulnerabilities in the notebooks.

Reply:
ok